website/docs/docs/cloud/manage-access/mfa.md

```diff
@@ -7,6 +7,13 @@ sidebar: null
 
 # Multi-factor authentication
 
+:::important
+
+
+dbt Cloud enforces multi-factor authentication (MFA) for all users with username and password credentials. Users who have not set up MFA will see a notification asking them to configure one of the supported methods when they log in. If they do not, they will have to configure MFA on the next login or be unable to access dbt Cloud.
```

Comment:
Will they need to write a support ticket to get unblocked? Suggest using second person for clarity. 
```suggestion
dbt Cloud enforces multi-factor authentication (MFA) when anyone signs in with username and password credentials. If MFA is not set up, you'll be asked to configure one of the supported methods when you log in. If you don't configure MFA after your second login and notification, you'll be unable to access dbt Cloud.
```

Reply:
There is no unblocked. They have to configure MFA at that point